UsingGoogleSignInButton.md

```diff
@@ -0,0 +1,44 @@
+# Using the Google Sign In Button
+
+Depending upon your platform, GoogleSignIn-iOS provides a few options to use
+when creating a sign-in button.
+
+## SwiftUI
+
+Creating a Google sign-in button in SwiftUI can be as simple as this:
+
+```
+GoogleSignInButton {
+  // Whatever action you'd like to take on tap
```

Comment:
Should we provide some pseudo code for extra context?

Reply:
I've added some surrounding code view-related code. Is that what you mean? IMHO, I think it's a little strange to have it for the SwiftUI section and not the others. Adding the same to the others feels like we're tutorializing how to add views to UIs in AppKit, UIKit, and SwiftUI, which isn't really the goal here. That's why I didn't have that context before.

Let me know what you think.

Reply:
I'm suggesting that we show how you would kick off a sign-in flow on tap.

Reply:
From offline conversation, I've added a call to `GIDSignIn.sharedInstance.signIn` in the action for the `GoogleSignInButton`.